### PR TITLE
feat(plugin): ImageCaption

### PR DESCRIPTION
**Pull request by MrDiamondDog**
A plugin that adds a /caption command that lets you add a caption to any image.

### CODE REVIEW

**Comment by MrDiamondDog** [comment]
Todo: 
 -[ ] add gif support
 -[ ] preview before sending

File: src/plugins/imageCaption.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            resolve(new File([blob], "caption.png"));
+        });
+    });
+}
+
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
```
Comment (TheKodeToad) [inline src/plugins/imageCaption.ts:65]: You don't need it again

Comment (MrDiamondDog) [reply src/plugins/imageCaption.ts:65]: no clue why it did that, will fix in next commit

**Comment by MrDiamondDog** [comment]
Why did it merge branch 7 times, sorry

**Comment by TheKodeToad** [comment]
It'll get squashed

**Pull request closed without merging**